service/history/execution/mutable_state_util.go

```diff
@@ -338,7 +338,7 @@ func CreatePersistenceMutableState(ms MutableState) *persistence.WorkflowMutable
 		SignalInfos:         signalInfos,
 		RequestCancelInfos:  cancellationInfos,
 		ChildExecutionInfos: childInfos,
-		ReplicationState:    replicationState,
+		VersionHistories:    versionHistories,
```

Comment:
Just curious. How did the code work previously without persisting this field?

Reply:
I think this method only used in test and test are mocking the replication state